teeracle/src/mock.rs

```diff
@@ -117,7 +129,7 @@ pub type Moment = u64;
 
 impl pallet_timestamp::Config for Test {
 	type Moment = Moment;
-	type OnTimestampSet = ();
+	type OnTimestampSet = Aura;
```

Comment:
Hmm, I see. This whole PR looks like an overkill, but I guess we can remove the feature flag now in our runtimes at least 🥳

Reply:
Uff, I see it is gone now: https://github.com/integritee-network/integritee-node/blob/1a18046482c72620898300e04b29fa7a72e7a3ff/runtime/src/lib.rs#L289

Here we used to have a feature flag that set `OnTimestamp= ()` in case we were running the benchmark because Aura didn't like it when we messed with time. But I guess your implementation now, is better than the runtime hack we did before anyhow.